### DISCUSSION TITLE
Open Chest Command

### DISCUSSION THREAD

**Discussion by pincoide** [Jukkales/NecroLens#9] [Ideas]
The auto open chest feels a bit dangerous (especially at high floors), so would it be possible to add a command like /openchest that tries to open the chest closest to you so you don't have to lose the current target or move around because you can't click the chest? 😄

**Comment by Jukkales** [comment]
Like that idea! 
im gonna implement that as command

**Comment by Jukkales** [comment]
Its up with 1.0.5.0 - feel free to give feedback :)

**Reply by pincoide** [reply]
nice, thanks!
I'll try it later and let you know if there are any issues :)

**Reply by pincoide** [reply]
I tried it for a while now, and it seems to be working well, there is only a tiny issue sometimes with accursed hoards where it doesn't see the chest for some reason (or targets something too far away), besides that is a great help :)

**Reply by Jukkales** [reply]
i tested around a bit and optimized how accursed hoards are handled. should work better now

**Reply by pincoide** [reply]
I've opened tons of chests (including hoards) and I had no more issues, thanks for the quick fix :)
